sui/its.js

```diff
@@ -18,6 +18,47 @@ function parseTrustedChains(config, trustedChain) {
     return trustedChain.split(',');
 }
 
+async function setFlowLimits(keypair, client, config, contracts, args, options) {
```

Comment:
add a test for set/unset `test-sui.yaml` for coverage. you'll need to use a snapshot release temporarily